tests/acceptance/08_commands/01_modules/module_file_test.cf

```diff
@@ -0,0 +1,59 @@
+body common control
+{
+        inputs => { "../../default.cf.sub" };
+        bundlesequence  => { default("$(this.promise_filename)") };
+        version => "1.0";
+}
+
+bundle agent init
+{
+  classes:
+      "ok_to_run" not => regcmp("/var/cfengine|.*\.cf-agent", "$(sys.workdir)");
```

Comment:
This doesn't look very protective for something that can leave behind files.

Reply:
I can leave it out. Just found in other tests.  So I assumed this is the way to go

Reply:
@nickanderson what do you think? I don't know why it's there.

Reply:
I don't know why it's there either.

Reply:
So i can just remove this check?

Reply:
Yes, please.